tcf_website/static/reviews/review.css

```diff
@@ -1,7 +1,21 @@
-.review .review-header {
-    color: var(--accent-color);
-    font-size: 1.25rem;
-}  
+.review-content .review-text.collapse:not(.show) {
+  display: block;
+  height: 12rem;
+  overflow: hidden;
+}
+.review-content .review-text.collapsing {
+  height: 12rem;
+}
+
+.review-content a.collapsed::after {
+  content: '...See More';
```

Comment:
Why is the text in the after as opposed to HTML? Does it have to do with Bootstrap

Reply:
No bootstrap, just a css thing. Will make a comment on it in the html